cicecore/cicedyn/general/ice_init.F90

```diff
@@ -260,6 +260,7 @@ subroutine input_data
         highfreq,       natmiter,        atmiter_conv,  calc_dragio,    &
         ustar_min,      emissivity,      iceruf,        iceruf_ocn,     &
         fbot_xfer_type, update_ocn_f,    l_mpond_fresh, tfrz_option,    &
+        saltflux_option, ice_ref_salinity, &
```

Comment:
ice_ref_salinity could be shifted left one space, align &.

Reply:
Ok